graphql/task_resolver.go

```diff
@@ -430,6 +431,16 @@ func (r *taskResolver) GeneratedByName(ctx context.Context, obj *restModel.APITa
 	return &name, nil
 }
 
+// ImageID is the resolver for the imageId field.
+func (r *taskResolver) ImageID(ctx context.Context, obj *restModel.APITask) (string, error) {
+	distroId := *obj.DistroId
+	imageId, err := distro.GetImageIdFromDistro(ctx, distroId)
```

Comment:
Let's use the ID directly:
```suggestion
	imageId, err := distro.GetImageIdFromDistro(ctx, *obj.DistroId)
```